Library/Homebrew/hardware.rb

```diff
@@ -7,28 +7,31 @@ class CPU
     INTEL_32BIT_ARCHS = [:i386].freeze
     INTEL_64BIT_ARCHS = [:x86_64].freeze
     PPC_32BIT_ARCHS   = [:ppc, :ppc32, :ppc7400, :ppc7450, :ppc970].freeze
-    PPC_64BIT_ARCHS   = [:ppc64].freeze
+    PPC_64BIT_ARCHS   = [:ppc64, :ppc64le, :ppc970].freeze
 
     class << self
       OPTIMIZATION_FLAGS = {
-        native:  "-march=native",
         nehalem: "-march=nehalem",
```

Comment:
```suggestion
        native:  arch_flag("native"),
        nehalem: "-march=nehalem",
```

Reply:
This won't work because this constant is defined at singleton class level.

Reply:
Yeh, of course. How about: get rid of the `OPTIMIZATION_FLAGS` constant (it's unused) and move it all into the `optimization_flags` flag method instead.

Reply:
If we want to call the class method `arch_flag`, this constant would need to be moved to class level